CMakeLists.txt

```diff
@@ -1,155 +1,143 @@
 ### CMakeList for MC Hydra
 
-cmake_minimum_required(VERSION 3.0)
-cmake_policy(SET CMP0054 OLD)
-
-# project name
-project(Hydra)
-
-# The version number.
-set (Hydra_VERSION_MAJOR 1)
-set (Hydra_VERSION_MINOR 0)
-set (Hydra_VERSION_PATCH 0)
-
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
- MESSAGE(STATUS "===> Compiling with Clang")
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
- MESSAGE(STATUS "===> Compiling with GCC")
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
- MESSAGE(STATUS "===> Compiling with Intel")
-elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
- MESSAGE(STATUS "===> Compiling with MSVC")
-endif()
```

Comment:
This is already printed by CMake's configure.

Reply:
Sure. I will just comment this out.  A piece of code  to place conditional commands depending the compiler.